__tests__/e2e/helpers/ai-agent.ts

```diff
@@ -141,25 +141,31 @@ type Grade = {
 	summary: string
 }
 
-export function checkToolUsage({
+export function getPrismicMcpTools({
```

Comment:
Instead of having a function that check, the utility can be a more generic, it give you the Prismic MCP server tools (unique set).

Then you can `expect` properly what you want. More flexible to test for one or multiple tool called.